shops/src/main/resources/shops-migrations/V2__Create_default_shopitems.sql

```diff
@@ -172,9 +172,23 @@ INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlo
-INSERT IGNORE INTO shopitems_dynamic_pricing VALUES (@shopItemId, 10, 25, 35, 35, 40, 45, 50000, 200000, 50000);
-INSERT IGNORE INTO shopitems_flags (shopItemId, PersistentKey, PersistentValue) VALUES (@shopItemId, 'IGNORE_MODELDATA', 'true');
-
-INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Fisherman', 'FISHING_ROD', 'Fishing Rod', 0, 21, 1, 1, 500, 0);
\ No newline at end of file
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Tradesman', 'COD', 'Fish', 0, 53, 1, 1, 40, 20);
+SELECT id INTO @codShopItemId FROM shopitems WHERE Shopkeeper = 'Tradesman' AND Material = 'COD';
+INSERT IGNORE INTO shopitems_dynamic_pricing VALUES (@codShopItemId, 10, 25, 35, 35, 40, 45, 50000, 200000, 50000);
+INSERT IGNORE INTO shopitems_flags (shopItemId, PersistentKey, PersistentValue) VALUES (@codShopItemId, 'IGNORE_MODELDATA', 'true');
+
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Tradesman', 'FISHING_ROD', 'Fishing Rod', 0, 12, 1, 1, 500, 0);
+
+-- Woodcutting
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Tradesman', 'DIAMOND_AXE', 'Power Axe', 0, 14, 1, 1, 16, 0);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Tradesman', 'GOLDEN_AXE', 'Booster Axe', 0, 23, 1, 1, 12, 0);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Tradesman', 'IRON_AXE', 'Iron Axe', 0, 32, 1, 1, 6, 0);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Tradesman', 'MANGROVE_PROPAGULE', 'Mangrove Propagule', 0, 15, 1, 1, 3, 0);
+SELECT id INTO @diamondAxeShopItemId FROM shopitems WHERE Shopkeeper = 'Tradesman' AND Material = 'DIAMOND_AXE';
+SELECT id INTO @goldenAxeShopItemId FROM shopitems WHERE Shopkeeper = 'Tradesman' AND Material = 'GOLDEN_AXE';
+SELECT id INTO @ironAxeShopItemId FROM shopitems WHERE Shopkeeper = 'Tradesman' AND Material = 'IRON_AXE';
+SELECT id INTO @saplingShopItemId FROM shopitems WHERE Shopkeeper = 'Tradesman' AND Material = 'MANGROVE_PROPAGULE';
+INSERT IGNORE INTO shopitems_flags (shopItemId, PersistentKey, PersistentValue) VALUES (@diamondAxeShopItemId, 'SHOP_CURRENCY', 'BARK');
+INSERT IGNORE INTO shopitems_flags (shopItemId, PersistentKey, PersistentValue) VALUES (@goldenAxeShopItemId, 'SHOP_CURRENCY', 'BARK');
+INSERT IGNORE INTO shopitems_flags (shopItemId, PersistentKey, PersistentValue) VALUES (@ironAxeShopItemId, 'SHOP_CURRENCY', 'BARK');
+INSERT IGNORE INTO shopitems_flags (shopItemId, PersistentKey, PersistentValue) VALUES (@saplingShopItemId , 'SHOP_CURRENCY', 'BARK');
```

Comment:
Instead of combining all the professions into a single vendor, I think its better if we keep them seperate and make one called Lumberjack or something, I have a shopkeeper model to use for that